snippets/HSLToRGB.md

```diff
@@ -0,0 +1,25 @@
+---
+title: HSLToRGB
+tags: math,intermediate
+---
+
+Converts a HSL color tuple to RGB format.
+
+- Use the [HSL to RGB conversion formula](https://en.wikipedia.org/wiki/HSL_and_HSV#HSL_to_RGB) to convert to the appropriate format.
+- The range of the input parameters is H: [0, 360], S: [0, 100], L: [0, 100].
+- The range of all output values is [0, 255].
```

Comment:
//

Reply:
I didn't understand that. Could you please clarify?